tutorials/workshop_examples/launch/turtlebot_slam.launch

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<launch>
+
+<arg name="record" default="False" doc="Record results in a bag file"/>
+<arg name="play" default="False" doc="Play recorded bag file"/>
+<arg name="bagfile" default="" doc="Play recorded bag file"/>
```

Comment:
Interesting, I did not know that we could add doc string to `<arg>` elements.

Reply:
I learned it recently as well. This documentation shows up when you do 'roslaunch launchfile.launch--ros-args'